wavenet/model.py

```diff
@@ -84,8 +84,17 @@ def __init__(self,
         self.initial_filter_width = initial_filter_width
         self.histograms = histograms
 
+        self.receptive_field = self._calculate_receptive_field()
         self.variables = self._create_variables()
 
+    def _calculate_receptive_field(self):
+        receptive_field = (self.filter_width - 1) * sum(self.dilations) + 1
+        if self.scalar_input:
+            receptive_field += self.initial_filter_width - 1
+        else:
+            receptive_field += self.filter_width - 1
+        return receptive_field
```

Comment:
Good points. Thanks for the explanation.

Rats. That leaves us without explanation for the test behavior.